dtests/src/test/java/io/snappydata/hydra/consistency/consistencyUsingSnappyJob.inc

```diff
@@ -0,0 +1,59 @@
+INCLUDE $JTESTS/io/snappydata/hydra/connectionPool/connPool.inc;
+
+INCLUDE $JTESTS/io/snappydata/hydra/consistency/schema_nonJDBC.inc;
+
+INCLUDE $JTESTS/io/snappydata/hydra/consistency/startSnappyCluster.inc;
+
+//INCLUDE $JTESTS/io/snappydata/hydra/testDMLOps/startDerby.inc;
+
+INITTASK     taskClass   = io.snappydata.hydra.consistency.SnappyConsistencyTest taskMethod  = HydraTask_initialize
+	threadGroups = snappyThreads,dmlThreads,selectThreads;
+
+INITTASK     taskClass   = io.snappydata.hydra.testDMLOps.SnappyDMLOpsUtil taskMethod  = HydraTask_initialize
+	threadGroups = snappyThreads,dmlThreads,selectThreads;
+
+INITTASK     taskClass   = io.snappydata.hydra.testDMLOps.SnappyDMLOpsUtil taskMethod  = HydraTask_createSnappySchemas
+	threadGroups = snappyThreads;
+
+INITTASK     taskClass   = io.snappydata.hydra.testDMLOps.SnappyDMLOpsUtil taskMethod  = HydraTask_createSnappyTables
+	threadGroups = snappyThreads;
+
+INITTASK     taskClass   = io.snappydata.hydra.testDMLOps.SnappyDMLOpsUtil taskMethod  = HydraTask_saveTableMetaDataToBB
+  threadGroups = snappyThreads;
+
+INITTASK     taskClass   = io.snappydata.hydra.consistency.SnappyConsistencyTest taskMethod  = HydraTask_registerDMLThreads
+  threadGroups = dmlThreads;
+
+//Load some data in the data, before starting any operations.
+INITTASK     taskClass   = io.snappydata.hydra.consistency.SnappyConsistencyTest taskMethod  = HydraTask_populateTables
+  io.snappydata.hydra.testDMLOps.SnappySchemaPrms-batchSize =1000000
```

Comment:
Have at least 5-10 million rows in column tables otherwise, some tables may not even have column batches in default cases.